.jvmopts

```diff
@@ -0,0 +1,4 @@
+-Xms512M
```

Comment:
Unfortunately this config should be removed as it interfere with our CI JVM settings.
To solve problems with OOME in tests I would advise to change sbt memory settings, maybe `sbt -mem 4096` is enought

Reply:
Sounds good, I've removed this file.